debug/src/devtools/hooks.js

```diff
@@ -0,0 +1,181 @@
+/* istanbul ignore file */
+import { options } from 'preact';
+import { USE_STATE, USE_CALLBACK } from 'preact/hooks/constants';
```

Comment:
This import is failing for me :thinking: 

Should it be `preact/hooks/src/constants` or `preact/hooks/dist/constants`?

Reply:
Also it's unclear to me why this is a separate import from the one below :point_down:

Reply:
probably

Reply:
Good catch, that was just an oversight and it should be merged with the other import :+1: